ui/components/ui/icon/icon-eye.js

```diff
@@ -6,17 +6,24 @@ const IconEye = ({
   color = 'currentColor',
   ariaLabel,
   className,
+  slash = false,
 }) => (
+  // This SVG adapted from `@fortawesome/fontawesome-free@5.13.0`.
+  // It is a combination of the SVGs `regular/eye.svg` and `regular/eye-slash.svg`.
   <svg
     width={size}
     height={size}
     fill={color}
     className={className}
     aria-label={ariaLabel}
     xmlns="http://www.w3.org/2000/svg"
-    viewBox="0 0 512 512"
+    viewBox="0 0 640 512"
```

Comment:
What is the purpose of changing the viewBox here?

Reply:
The fontawesome SVGs each used a different viewbox. Both were an increase from the Figma version seemingly, but they had different widths (the eye-slash one was slightly wider), so I went with the wider one to ensure it didn't get cut off.

As we just discussed I'm going to move this to a separate component, so I'll restore the exact viewbox that the SVGs use.